avocado/plugins/yaml_to_mux.py

```diff
@@ -261,9 +261,9 @@ def run(self, args):
             debug = getattr(args, "mux_debug", False)
             try:
                 args.mux.data_merge(create_from_yaml(multiplex_files, debug))
-            except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+            except IOError:
+                msg = 'Cannot access the multiplex file(s) in this host.'
+                logging.getLogger("avocado.app").warning(msg)
```

Comment:
Should that failure be so forgiving? It seems to me that under both circustances (local and remote) this should error out more explicitly.